src/ColorVectorSpace.jl

```diff
@@ -314,6 +314,10 @@ end
 (.*){T<:Number}(A::AbstractArray{T}, b::TransparentGray) = mul(A, b)
 (.*){T<:Number}(b::TransparentGray, A::AbstractArray{T}) = mul(b, A)
 
+varm{C<:AbstractGray}(v::AbstractArray{C}, s::AbstractGray; corrected::Bool=true) =
+        varm(map(gray,v),gray(s); corrected=corrected)
+real{C<:AbstractGray}(x::Type{C}) = C
```

Comment:
Do you need this (`real`)? If I'm reading correctly, it takes a value input and returns a type. In contrast,
```jl
julia> real(3+4im)
3
```
You could call it `real_type`, or use `real(typeof(x))`:
```jl
julia> real(Complex{Float32})
Float32
```